src/baseevents.h

```diff
@@ -44,7 +44,15 @@ class Event
 			return scripted;
 		}
 
+		bool isFromLua() const {
+			return fromLua;
+		}
+		void setFromLua(bool b) {
+			fromLua = b;
+		}
+
 		bool scripted = false;
+		bool fromLua = false;
```

Comment:
No need for `get`/`set` if the attribute is public and the methods are basic.

Reply:
will do that